test/cqlpy/test_protocol_exceptions.py

```diff
@@ -58,35 +59,53 @@ def try_connect(host, port, creds, protocol_version):
     with cql_with_protocol(host, port, creds, protocol_version) as session:
         return 1 if session else 0
 
+@contextmanager
+def debug_exceptions_logging(cql):
+    debug_enabled = False
+    try:
+        try:
+            nodetool.setlogginglevel(cql, "exception", "debug")
+            debug_enabled = True
+        except Exception as exc:
+            print(f"Failed to enable debug logging for exception logger: {exc}")
+        yield
+    finally:
+        if debug_enabled:
+            try:
+                nodetool.setlogginglevel(cql, "exception", "info")
```

Comment:
This is not great - how do we know "info" was the previously set default? Maybe it's always the default in Scylla?
 What if  A better approach would have been to use the REST API to read the old setting and restore it later.